packages/playwright/src/prompt.ts

```diff
@@ -38,16 +38,20 @@ export async function attachErrorPrompts(testInfo: TestInfo, sourceCache: Map<st
     }
   }
 
-  for (const [index, error] of testInfo.errors.entries()) {
+  const errors = [...testInfo.errors.entries()].filter(([index, error]) => {
     if (!error.message)
-      return;
+      return false;
     if (testInfo.attachments.find(a => a.name === `_prompt-${index}`))
```

Comment:
I think this filter condition conflicts with checking for `errors.length === 1` below.

Reply:
good catch! fixed in https://github.com/microsoft/playwright/pull/35341/commits/cf6e070abce82acd7114621fe13584eec026d12d